packages/tasks/src/tasks/any-to-any/data.ts

```diff
@@ -0,0 +1,55 @@
+import type { TaskDataCustom } from "../index.js";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "All-modality alignment dataset with fine-grained preference annotations and language feedback.",
+			id: "PKU-Alignment/align-anything-400k",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "distractedboyfriendmeme.jpeg",
```

Comment:
can you open a PR to this repo https://huggingface.co/datasets/huggingfacejs/tasks/tree/main under any-to-any folder for this file?